### PR TITLE
updated fleet registration for openshift clusters

### PR DESCRIPTION
**Pull request by aryan16**
Added an extra parameter while creating membership for openshift cluster.

### CODE REVIEW

**Comment by ZhengzheYang** [comment]
Code LGTM, waiting for @zerobfd to see if we should merge this since asmcli is in maintenance mode.

File: asmcli/lib/context.sh
Comment (zerobfd) [inline asmcli/lib/context.sh:68]: Can we name this variable something more general? e.g. `PRIVATE_ISSUER` or `USE_JWKS_NOT_OIDC` or `NO_PUBLIC_ISSUER` or something. It doesn't fundamentally have anything to do with Openshift.

**Review by zerobfd — CHANGES_REQUESTED** [review]

File: asmcli/lib/context.sh
Comment (aryan16) [reply asmcli/lib/context.sh:68]: yeah sounds good, I used that name because it is only being used for openshift now. Good suggestion, will make the change.

**Review by zerobfd — APPROVED** [review]

**Pull request merged**
